src/components/Main/App.jsx

```diff
@@ -195,9 +195,9 @@ class App extends React.Component {
                           className="instructions mobile-hidden">
                           {i18n.t('app.instructions')}
                         </button>
-                        <Show when={this.props.authenticated}>
+                        <span>
```

Comment:
Do we actually need these `span` tags or are they only here to satisfy the snapshot tests? If it's only for the snapshot tests, I suggest we remove them.

Reply:
I pulled it in because that's what Show added, but looks like it doesn't buy us anything so I've removed it.